Suggesting few changes to the document (#25435)
grafana/grafana: docs/sources/panels/visualizations/heatmap.md

### Color scale

By default, Grafana calculates cell colors based on minimum and maximum buckets values. With Min and Max you can overwrite those values. Think of a bucket value as a Z-axis and Min and Max as Z-Min and Z-Max respectively.

> - **Min -** Minimum value using for cell color calculation. If bucket value is lesser than Min, it will be mapped to the "minimum" color. Default is series min value.

Reviewer: ```suggestion
- **Min -** Minimum value using for cell color calculation. If the bucket value is less than Min, then it is mapped to the "minimum" color. Default is series min value.
```

> - **Max -** Maximum value using for cell color calculation. If bucket value is greater than Max, it will be mapped to the "maximum" color. Default is series max value.

Reviewer: ```suggestion
- **Max -** Maximum value using for cell color calculation. If the bucket value is greater than Max, then it is mapped to the "maximum" color. Default is series max value.
```
